coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/jpa/ItemManager.java

```
@@ -88,6 +100,29 @@ public synchronized String persist(Item item) throws CycleIntroductionException
 		return id;
 	}
 	
+	public synchronized boolean persistAll(List<Item> items) throws CycleIntroductionException {
+		boolean worked = false;
+		EntityTransaction transaction = em.getTransaction();
+		try {
+			transaction.begin();
+			for (Item item : items) {
+				persistItem(item);
+			}
+			transaction.commit();
+			worked = true;
+		} catch (Exception ex) {
+			log.debug("Exception during save", ex);
+			if (transaction.isActive()) {
+				transaction.rollback();
+			}
+			if (ex instanceof CycleIntroductionException) {
```

Do you not want to throw an exception regardless?